Documentation/faq.md

```diff
@@ -2,6 +2,14 @@
 
 ### etcd, general
 
+#### When to use etcd
+
+etcd is designed to store a small amount of metadata in consistent, fault-tolerant way. Distributed systems use etcd as the root of scheduling and service discovery configuration storage. [Distributed lock][etcd-concurrency] is implemented on top of etcd, useful for master election in other distributed databases.
+
+#### When not to use etcd (etcd constraints)
+
+etcd can only handle small chunks data: configuration file, JSON, YAML, etc. etcd limits the size of request in 1.5MB; see [`maxRequestBytes`][max-request-limit]. And default storage size limit is 2GB, configurable with `--quota-backend-bytes` flag; supports up to 8GB, see [`DefaultQuotaBytes`][max-storage-limit].
```

Comment:
`etcd limits the size of request in 1.5MB` => `etcd limits the size of request to 1.5MB`